### PR TITLE
Feature/characteristic source

### PR DESCRIPTION
**Pull request by klngwll**
Fixing source which ends up on Instance instead of digitalCharacteristic, videoCharacteristic, projectionCharacteristic or soundCharacteristic.

Open for discussion.

### CODE REVIEW

**Comment by klngwll** [comment]
256 has been slightly remodeled. Some fields are no longer valid. This is only halfway to a real solution but still much better than the current situation. It could be a good thing to merge it and analyze it in a test environment while we still have some time to reverse or reshape until next release.

File: whelk-core/src/main/resources/ext/marcframe.json [hunk 1]
```diff
@@ -6816,10 +6816,37 @@
     },
 
     "256": {
+      "NOTE:LC": "Prior to the definition of this field in 1987, file characteristics were recorded in subfield $a of field 300",
       "aboutEntity": "?thing",
-      "$a": {"link": "digitalCharacteristic", "resourceType": "FileType", "property": "label"}
+      "addLink": "digitalCharacteristic",
+      "resourceType": "DigitalCharacteristic",
+      "repeatable": false,
+      "$a": {"addLink": "hasNote", "resourceType": "Note", "property": "label", "NOTE:repeatable": false},
```
Comment (niklasl) [inline whelk-core/src/main/resources/ext/marcframe.json]: I'd really prefer for this to either A) be reduced to just a `comment` property, or B) for this "sketchy" `digitalCharacteristic` to be changed to just a `hasNote` with type `marc:FileTypeNote`.

(Although the majority of data values seem to be a textual label for the `FileType` (somewhat in favour of the original mapping) I presume bib 256 is considered as legacy data which we might even drop the majority of?)

**Comment by niklasl** [comment]
I like this compromise. A total cleanup/normalization isn't feasible, and this is a solid step towards a more managable structure. Let's just sort out some minutia and then merge.

File: whelk-core/src/main/resources/ext/marcframe.json (hunk 1 above)
Comment (klngwll) [reply whelk-core/src/main/resources/ext/marcframe.json]: It has now been updated per discussion.

**Pull request merged**